### ISSUE TITLE
Ability to render to different audio codecs

### ISSUE THREAD

**Issue by gabeklavans** [gridsound/daw#68]
WAV is great cause it's uncompressed and can be converted to any other format, but it would be pretty nice to bounce directly to a smaller format like MP3 or AAC.

**Comment by mr21** [comment]
I have to confess that it's also mainly because it's the simpler way :D i don't know exactly the license about MP3 etc. so gridsound will probably have their free equivalent like flac or ogg.

**Comment by gabeklavans** [comment]
Yeah totally understandable to dodge MP3 cause of the licensing. But yeah, it would just be a nice feature for the future!

**Comment by mwaitzman** [comment]
MP3 has licensing??

I would suggest possibly suggesting that people go to https://www.conversion-tool.com/category/audio-converters/ (which has the ability to convert a selected file to MANY different types of audio files) if they want to convert their files to different formats. This would be a great option while you work on adding ogg/flac support, and might be a good idea to keep even later. Maybe you could even contact the owners of conversion-tool.com directly to see what libraries or w/e they used to facilitate the conversions
